### ISSUE TITLE
Silktide Feb 2023 findings

### ISSUE THREAD

**Issue by chrisrhymes** [FutureNorthants/northants-design-system#339]
- [ ] Custom Search component's for on label and id on input are not displaying correctly
- [ ] No meaningful colour contrast for footer links focus state
- [ ] Insufficient colour contrast for Latest news sort by
- [ ] Insufficient colour contrast for pagination focus state
- [ ] Header logo North theme insufficient contrast on hover state
- [ ] Section links with image (Country parks page) insufficient pixels changing when focused
